.circleci/config.yml

```diff
@@ -365,193 +408,145 @@ jobs:
     docker:
       - image: "circleci/node:12"
     resource_class: xlarge
-    environment:
-      CI_NAME: circleci
-      NUM_PAGES: "512"
     steps:
-      - checkout
-      - run:
-          command: npm install
-          working_directory: benchmarks/markdown_id
-      - run:
-          command: npm run build
+      - run-benchmark:
           working_directory: benchmarks/markdown_id
+          NUM_PAGES: "512"
+          BENCHMARK_CONTENT_SOURCE: MDX
```

Comment:
I'm not sure if there is distinction needed - but technically this benchmark is `MD` and not `MDX` (and similarly all others)

Reply:
This relates to an enumeration that we have in cloud where the values is `MDX`.